benchpress/qiskit_gym/manipulate/test_manipulate.py

```diff
@@ -137,6 +137,30 @@ def result():
         assert result
 
 
+    def test_random_clifford_decompose(self, benchmark):
+        """Decompose a random clifford into
+        basis [rz, sx, x, cz]
+        """
+        translate = generate_preset_pass_manager(
+            1, basis_gates=["rz", "sx", "x", "cz"]
```

Comment:
Can this be rewritten to use the basis set in the `default.conf`?

Reply:
I see that this line also appears in other tests like in `test_QV100_basis_change`:
https://github.com/Qiskit/benchpress/blob/27b978f887d2187827c7792187df7f0d364b7114/benchpress/qiskit_gym/manipulate/test_manipulate.py#L124

Reply:
yes, but it will be removed there as well when I get a chance

Reply:
I'm not sure how to do this. it's probably better to make a separate PR for all the `basis_gates` in this file, as well as for the `optimization_level`